packages/firestore/test/unit/local/persistence_test_helpers.ts

```diff
@@ -44,3 +55,42 @@ export async function testMemoryPersistence(): Promise<MemoryPersistence> {
   await persistence.start();
   return persistence;
 }
+
+/**
+ * Creates LocalStorageNotificationChannel instance for testing, destroying any
+ * previous contents if they existed.
+ */
+export async function testLocalStorageNotificationChannel(
+  ownerId: string,
+  existingMutationBatches: BatchId[],
+  existingQueryTargets: TargetId[]
+): Promise<TabNotificationChannel> {
+  window.localStorage.clear();
+  const instances = [];
+
+  if (existingMutationBatches.length + existingQueryTargets.length != 0) {
```

Comment:
Would `existingMutatitonBatches.length > 0 || existingQueryTargets.length > 0` be a more natural way to express this?

(for a brief moment I actually thought this was checking if the two arrays cancelled each other out or something... but that's silly since length can't be negative)

Reply:
Certainly :) Updated.